packages/rich-text-editor/src/components/ToggleIconButton/ToggleIconButton.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react"
+import { Tooltip } from "@kaizen/draft-tooltip"
+import { Icon } from "@kaizen/component-library"
+import classnames from "classnames"
+import styles from "./ToggleIconButton.scss"
+
+export interface ToggleIconButtonProps
+  extends React.ButtonHTMLAttributes<HTMLButtonElement> {
+  icon: React.SVGAttributes<SVGSymbolElement>
+  label: string
+  /*
+   * determines the active or inactive state along with the "aria-pressed" attribute
+   */
+  isActive?: boolean
+}
+
+export const ToggleIconButton: React.VFC<ToggleIconButtonProps> =
```

Comment:
Just saw the file structure too. Looking at our docs, it should be:
`src/RichTextEditor/components/ToggleIconButton`
as the `components` folder is only necessary for when you have subcomponents.

Reply:
Mind if I get a sense check from the both of you @HeartSquared @dougmacknz before I push this change:

![Screen Shot 2022-03-04 at 3 35 02 pm](https://user-images.githubusercontent.com/36558508/156700120-c34b923d-d3af-458d-8153-91e7e057e471.png)

Reply:
Yep that looks right! :)